proximal/examples/test_deconv_sinv_psf.py

```diff
@@ -44,19 +58,19 @@ def my_factory(psf, im_shape):
     
     def forward(im_src, im_sim):
         assert np.all(im_src.shape == im_sim.shape)
-        im_sim = fftconvolve(psf, im_src, mode='same')
+        im_sim = convolve(psf, im_src, mode='same', method='auto')
```

Comment:
The option `method='auto'` let Scipy to pick between fft convolution or shift-and-add convolution algorithms. Otherwise, it take too much time to solve the problem on my computer.